.travis.yml

```diff
@@ -1,11 +1,16 @@
 language: node_js
 node_js:
   - '6'
+  - '8'
 sudo: false
+before_install:
+  - npm install -g grunt-cli
+env:
+  - MONGODB_VERSION=2.4
+  - MONGODB_VERSION=3.2
 services:
   - mongodb
 before_script:
   - mongo admin --eval 'db.createUser({user:"admin",pwd:"admin",roles:[{role:"userAdminAnyDatabase",db:"admin"}]});'
 script:
   - grunt
-  - grunt coverallsio
```

Comment:
Is there a reason for removing this script

Reply:
never mind in the WHAT

Reply:
@grdryn asked to remove it since it is just to do a report on the test coverage of the project and takes too long to be done in the CI process. Also, it is not used for us at all.

Reply:
Also it was what broke the build when Node.js 8 was added, and I don't imagine anyone looks at the coveralls reports, so it didn't seem worthwhile spending time trying to fix that.

Reply:
I could fixed and also I could check that it has been used to don't allow create new things and merge without a test to cover it. So, I think is good we keep it.